backend/tests/test_integration_import.py

```
@@ -188,13 +197,15 @@ def test_preview_rejects_range_over_365_days(self, client):
         assert resp.status_code == 422
         assert "365" in resp.json()["detail"]
 
+    @pytest.mark.integration
     def test_preview_accepts_range_at_365_days(self, client, mock_schwab):
```

Ruff (0.15.13)

[warning] Unused method argument: `mock_schwab`

(ARG002)